pom.xml

```diff
@@ -174,5 +174,27 @@
 
         </dependencies>
     </dependencyManagement>
+    <!-- things to do if running on JDK 17 -->
+    <profiles>
+        <profile>
+            <id>java17-settings</id>
+            <activation>
+                <jdk>[17,)</jdk>
+            </activation>
+            <properties>
+                <javadoc.opts>-Xdoclint:none</javadoc.opts>
+            </properties>
+            <modules>
+                <module>rocker-runtime</module>
+                <module>rocker-compiler</module>
+                <module>rocker-maven-plugin</module>
+                <module>rocker-gradle-plugin</module>
+                <module>rocker-test-template</module>
+                <module>rocker-test-template-java-17</module>
```

Comment:
Can we rename this to rocker-test-template17?

Reply:
done